README.md

```diff
@@ -950,7 +950,37 @@ Do not call this API with the service-operator-access plan credentials.
                                 
 <b>Attention: **Use this option only for cleanup purposes for a cluster that's no longer available.** Applying it to an active and available cluster may result in unintended resource leftovers in your cluster.</b>
 
+### I can see my service instance in SAP BTP, but not its corresponding custom resource in a cluster. What can I do?
 
+Let's break down how to recover your Kubernetes custom resource that exists in SAP BTP but not in your Kubernetes cluster:
+
+#### Background
+
+The service instance in SAP BTP acts as the source of truth for your CR's configuration and state. By recreating the CR in Kubernetes with the exact same name, namespace, and cluster ID, you do not trigger the provisioning of a new service instance, but essentially re-establish the link between the Kubernetes representation and its existing counterpart in SAP BTP.
+
+#### Steps
+
+1. Retrieve CR Details:
+
+a) Access the service instance representing your CR in SAP BTP.
+
+b) Obtain the following details from the service instance:
+
+- The name of the custom resource.
+- The Kubernetes namespace where the CR should reside.
+- If applicable, retrieve the cluster ID or context associated with the CR in SAP BTP. This ensures the CR is recreated in the correct environment.
```

Comment:
remove

Reply:
@kerenlahav  I assume you are talking about removing the third one.